moma_mission/config/state_machine/piloting.yaml

```diff
@@ -45,21 +45,29 @@ HOME_ROBOT_FINAL:
 
 OBSERVATION_POSE:
   default_outcome: None
-  object_position_estimate: [0.8, 0.0, 0.0]
-  phi_range: [0, 0.4]
+  sample_name: observation
+  map_frame: world
+  object_frame: valve_gt
+  camera_frame: hand_eye_color_optical_frame
+  controlled_frame: tool_frame
+  object_size: 0.20
+  camera_info_topic: "/hand_eye/color/camera_info"
```

Comment:
This should probably be `/camera_info` which is our own "corrected" calibration.